src/sentry/integrations/slack/notify_action.py

```diff
@@ -52,6 +52,20 @@ def clean(self):
             self._pending_save = True
             return cleaned_data
 
+        if channel_prefix is None:
```

Comment:
@KKaempen So the only way `channel_prefix` is None is if we get the duplicate user match? I'm a bit confused this is how this case is conveyed.

Reply:
Yes, that's the only case where that'll be None. I was having trouble thinking of a better way of communicating a different failure case than the already-existing one. If you have any ideas I'm all ears.

Reply:
@KKaempen Another alternative is to throw a specific exception and catch it at a higher level. I should point out that we call `get_channel_id_with_timeout` in `find_channel_id_for_rule` and I think that's going to blow up if you have `None` as the prefix.

Reply:
Hmm, alright. I'll try to make that happen then.

Reply:
@scefali If I do that, I'll have to change where I test the bad behavior. Should I do something like 
```
try: 
    (bad call to function) 
    assert(False)
except: 
    assert True
```
?

Reply:
@KKaempen Yea, that works or you could try using `with pytest.raises`. There are some. examples in tests.

Reply:
Gotcha. Sounds cleaner, I'll look into that.